primitives/runtime-interface/proc-macro/src/runtime_interface/host_function_interface.rs

```diff
@@ -323,10 +330,19 @@ fn generate_host_function_implementation(
 		});
 	}
 
+	let cfg_attrs: Vec<_> = method
+		.attrs
+		.iter()
+		.filter(|a| a.path().is_ident("cfg"))
+		.map(|a| a.clone())
```

Comment:
```suggestion
		.cloned()
```

(I *think* this should compile, but please check to be sure.)